src/main/java/org/zendesk/client/v2/model/events/AgentMacroReferenceEvent.java

```diff
@@ -0,0 +1,46 @@
+package org.zendesk.client.v2.model.events;
+
+import org.zendesk.client.v2.model.Via;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class AgentMacroReferenceEvent extends Event {
+
+    private static final long serialVersionUID = 1L;
+
+    private Long macroId;
```

Comment:
I'm hesitant to set this as a String (which is what is used in the JSON), I'll submit a few Zendesk Support cases tomorrow for some documentation from Zendesk.